.changes/nextrelease/presign_additional_headers.json

```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "type": "enhancement",

```

Comment:
This should be marked as a `feature` in the Changelog Document, not just an `enhancement`.